pkg/cmd/server/origin/asset.go

```diff
@@ -27,6 +27,7 @@ import (
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 	"github.com/openshift/origin/pkg/cmd/server/crypto"
 	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+	oauthutil "github.com/openshift/origin/pkg/oauth/util"
```

Comment:
This package dependency was previously excluded intentionally.  The asset server should not depend on the oauth package at all. The two are logically independent.